topics/fair/tutorials/fair-ena/tutorial.md

```diff
@@ -0,0 +1,305 @@
+- Use cURL to submit read fastq files
+requirements:
+- type: none
+  title: "Web browser"
+- type: none
+  title: "A linux-based machine or linux emulator"
+- type: none
+  title: "Read data in fastq format"
+tags:
+- FAIR
+- ENA
+priority: 1
+contributions:
+  authorship:
+    - sonalhenson
+  editing:
+    - kkamieniecka
+    - smza
+  funding:
+    - ELIXIR-UK-DaSH
+subtopic: fair-data
+
+---
+> <agenda-title></agenda-title>
+>
+> In this tutorial we will deal with:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# 01. Introduction
+DNA sequencing has become one of the key technologies in molecular biology, with applications in diagnostics, evolutionary biology, drug discovery, forensics and much more. Drop in sequencing costs and breakthroughs in sequencing technologies has seen increasing utilization of sequencing as a research tool, featuring in thousands of life-science publications every year. 
+
+Prior to publication many journals and funders require authors to submit their raw sequence data to one of the three INSDC member databases – **ENA, NCBI or DDBJ** – between which data is synchronised on a daily basis. INSDC is the core infrastructure for sharing nucleotide sequence data and metadata in the public domain. Data in INSDC member databases is available permanently, for free and with unrestricted access. For each submitted sequence a unique accession number is issued which can be reported in the publication. 
+
+The three databases have different methods for making submissions. If your database of choice is ENA and you 
+need to submit data stored on a remote server, you are in the right place. This tutorial will cover how to find your way around the ENA Webin portal for uploading raw sequencing read data as well as accompanying metadata, and use [cURL](https://en.wikipedia.org/wiki/CURL) to copy read files over to ENA's FTP server. 
+
+If you would like to use Galaxy tools for submission to ENA you may find [Submitting sequence data to ENA]( {% link topics/galaxy-interface/tutorials/upload-data-to-ena/tutorial.md %}) tutorial helpful. 
+
+> <tip-title>What is cURL?</tip-title>
+> cURL is a command-line tool and library for transferring data over the internet. It allows you to send and receive data from various protocols like HTTP, FTP, and more. In simple terms, it's a tool that helps your computer talk to other computers on the internet and fetch or send information, like downloading files from a website or making API requests. 
+{: .tip}
+
+## ENA Submission Routes
+
+There are three routes via which one can submit data to ENA. 
+1.	The Webin Interactive portal allows you to submit data by filling out forms on your browser. This is what we will cover in this tutorial,
+2.	Webin-command-line interface (CLI), which allows submissions through a terminal, and
+3.	Programmatic submission, which requires the submissions to be prepared as XML documents and then uploaded using cURL or Webin portal. 
+
+**Table 1.** ENA submission routes

```

Comment:
we should make this easier, https://github.com/galaxyproject/training-material/issues/4460 but I guess it will take a while.

Reply:
Hi @hexylena, Project 19 at the ELIXIR BioHackathon is trying to make this easier through Galaxy. We have been discussing already preparing a GTN tutorial on the updated ENA Upload tool and being built Interactive Metadata builder tool.
Although I'm talking ENA uploads in Galaxy, and I see you link is to image support.

Reply:
> Preparing a GTN tutorial to use the ENA upload tool

Ah, would be nice to have any relevant updates to https://training.galaxyproject.org/training-material/topics/galaxy-interface/tutorials/upload-data-to-ena/tutorial.html !

Reply:
Agreed 100! This has been discussed this week. We are refreshing the language in the tool and also the new tool will help with some stability between ENA and Galaxy, and therefore on GTN content. So short answer, Yes. The plan is to review tutorial after the tools are reviewed.